src/row.rs

```diff
@@ -75,21 +75,21 @@ pub(crate) fn join_column_names<R: Row>() -> Option<String> {
 
 #[cfg(test)]
 mod tests {
-    // XXX: need for `derive(Row)`. Provide `row(crate = ..)` instead.
-    use crate as clickhouse;
-    use clickhouse::Row;
+    use crate::Row;
 
     use super::*;
 
     #[test]
     fn it_grabs_simple_struct() {
         #[derive(Row)]
+        #[row(crate = crate)]
```

Comment:
Couple of design nits:

* `row` is a little ambiguous of a namespace for an attribute. It's quite conceivable that this could collide with other implementations. We discussed it and we'd like to change this to `clickhouse`.
* Serde's equivalent of this attribute expects the path to be quoted: https://serde.rs/container-attrs.html#crate
   While this works as-written, we'd like to avoid surprising users by having different semantics here.

I'll be opening a PR using this one as a basis so you still get attribution, but with these changes and merge conflicts addressed.